doc-content/jbpm-docs/src/main/asciidoc/ReleaseNotes/Release.7.17.0.Final-section.adoc

```diff
@@ -26,3 +26,48 @@ Furthermore, it is also possible for users to interactively select nodes in the
 on the left for a more specific visualization of the node data.
 
 image:ReleaseNotes/process_admin_operations_7.17.0.png[align="center", title="Process instance diagram"]
+
+== Service task repository integrated with Business Central
+
+Service task repository (aka work item repository) has been integrated into Business Central. It provides two areas
+to manage service tasks
+
+* Global administration - accessible via settings page with dedicated `Service Task Administration` tile
+* Project level management - accessible via project settings
+
+Global administration allows process administrators to define what service tasks can be used within projects
+by either enabling or disabling them. Most of service tasks available in repository are disabled by
+default except those that come with jBPM out of the box e.g. REST, Email, Decision, Business rule.
+
+Administrators can also upload custom service tasks to make them available for projects.
+
+Project level management allows to install given service task into the project. Installation consists of
+
+* updating project dependencies (modifies pom.xml of the project)
+* updating deployment descriptor - registers work item handler for the service task
+* installs icon and work definition file into project resources (global area)
+
+To learn more read http://mswiderski.blogspot.com/2019/01/service-task-repository-integrated-into.html[this article]
+
+== Docker service task
+
+jbpm work items have been equipped to provide several service tasks to interact with docker
+
+* Create container
+* Inspect container
+* Kill container
+* List containers
+* Start container
+* Stop container
+* List images
+
+These can be used directly by taking advantage of service task repository integrated into Business Central.
+
+== VS Code Extension for jBPM Business Applications
+
+If you are developing your apps using Visual Studio Code you can now install a new
+https://marketplace.visualstudio.com/items?itemName=tsurdilovic.jbavsc[jBPM Business Application extension].
+With this extension and the great tooling support of VSC you can now generate, develop,
```

Comment:
delete great
apps  applications. The website calls them business applications so always spell out.

Reply:
Watch this video for a demonstration.